api/src/routes/HttpRoutes.js

```diff
@@ -324,6 +326,14 @@ restify.serve(router, Statement, {
     return;
   },
   preUpdate: (req, res) => res.sendStatus(405),
+  postDelete: (req, _, next) => {
```

Comment:
For consistency going forward, do you think we should make this an async function? We could wrap the contents in a try-catch block to continue calling next with errors. Then we could await updateStatementCountsInOrg and call next after.